packages/cloudflare-worker-mock/src/index.ts

```diff
@@ -99,4 +101,77 @@ export function makeCloudflareWorkerKVEnv(
   return cloudflareWorkerKVEnv;
 }
 
+/**
+ * Create a mock Request for a Cloudflare Worker
+ *
+ * A CloudflareRequestAttributes object can be created manually, but it requires
+ * the developer to specify every field. This helper function allows the
+ * developer to specify only fields required by tests.
+ */
+export function makeCloudflareWorkerRequest(
+  input: RequestInfo,
+  init?: MockCloudflareRequestInit,
+): Request {
+  if (init == null) {
+    return new Request(input);
+  }
+
+  const attr = init.cf;
+  const tlsAttr = attr.tlsClientAuth ? attr.tlsClientAuth : {};
+  const cf: CloudflareRequestAttributes = {
+    asn: attr.asn ? attr.asn : '395747',
+    city: attr.city ? attr.city : 'Austin',
+    colo: attr.colo ? attr.colo : 'AUS',
+    continent: attr.continent ? attr.continent : 'NA',
+    country: attr.country ? attr.country : 'US',
+    exclusive: attr.exclusive ? attr.exclusive : '0',
+    group: attr.exclusive ? attr.exclusive : '0',
+    'group-weight': attr['group-weight'] ? attr['group-weight'] : '0',
+    httpProtocol: attr.httpProtocol ? attr.httpProtocol : 'HTTP/2',
+    latitude: attr.latitude ? attr.latitude : 30.2713,
+    longitude: attr.longitude ? attr.longitude : -97.7426,
+    postalCode: attr.postalCode ? attr.postalCode : '78701',
```

Comment:
My zip is `78745` actually 😉

Reply:
:laughing:  These are the examples directly from the Cloudflare documentation.